cli/bpbuild/git.go

```diff
@@ -0,0 +1,36 @@
+package bpbuild
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/go-git/go-git/v5"
+)
+
+const defaultRemote = "origin"
+
+// getRepoName finds upstream repo name from a given repo directory
+func getRepoName(dir string) (string, error) {
+	r, err := git.PlainOpen(dir)
+	if err != nil {
+		return "", fmt.Errorf("error opening git dir %s: %w", dir, err)
+	}
+	rm, err := r.Remote(defaultRemote)
+	if err != nil {
+		return "", fmt.Errorf("error finding remote %s in git dir %s: %w", defaultRemote, dir, err)
+	}
+
+	// validate remote URL
+	remoteURL, err := url.Parse(rm.Config().URLs[0])
+	if err != nil {
+		return "", fmt.Errorf("error parsing remote URL: %w", err)
+	}
+	repoOwnerName := strings.TrimSuffix(remoteURL.Path, "/")
+	splitRepoOwnerName := strings.Split(repoOwnerName, "/")
```

Comment:
nit: can you call this `splitRepoOwnerName` as `trimmedRemoteUrl` since it has both repo and owner name. The variable appears to suggest it's just the repo owner name that is split.

Reply:
Good point, changed to `trimmedRemotePath` since it is just path without host.